components/ChooseSong.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import { Container, Row, Col, Table, Form } from 'react-bootstrap/'
+import PropTypes from 'prop-types'
+import { searchSpotifyTracks } from '../utils/fetcher'
+import useAuth from '../utils/useAuth'
+const AddMemory = () => {
+  const [, token] = useAuth()
+  const [song, setSong] = useState('')
+  const [searchResults, setSearchResults] = useState()
+
+  useEffect(() => {
+    const searchSpotify = async () => {
+      if (song.length > 2) {
+        try {
+          const response = await searchSpotifyTracks(song, 5, token)
+          setSearchResults(response)
+        } catch (error) {
+          console.log(error)
+        }
+      }
+    }
+    if (token) {
+      searchSpotify()
+    }
+  }, [song, token])
+  return (<>
```

Comment:
Add a space between useEffect and the return

Reply:
Plus you don't need the fragment if it's all wrapped in one html tag (the Container)

Reply:
Nice!